templates/todo/projects/nodejs-mongo-aks/src/web/manifests/deployment.tmpl.yaml

```diff
@@ -18,11 +18,11 @@ spec:
           ports:
             - containerPort: 3000
           env:
-            - name: REACT_APP_API_BASE_URL
+            - name: VITE_API_BASE_URL
               value: /api
-            - name: REACT_APP_APPLICATIONINSIGHTS_CONNECTION_STRING
+            - name: VITE_APPLICATIONINSIGHTS_CONNECTION_STRING
               valueFrom:
                 configMapKeyRef:
                   name: todo-web-config
-                  key: REACT_APP_APPLICATIONINSIGHTS_CONNECTION_STRING
+                  key: VITE_APPLICATIONINSIGHTS_CONNECTION_STRING
```

Comment:
Is this still required in the env?   The settings are fused to the web app during build, it does not need any env in the host